frontend/src/scenes/onboarding/sdks/sdk-install-instructions/ios.tsx

```diff
@@ -27,11 +27,10 @@ import UIKit
 
 class AppDelegate: NSObject, UIApplicationDelegate {
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
-        let POSTHOG_API_KEY = "<ph_project_api_key>"
-        let POSTHOG_HOST = "<ph_instance_address>"
+        let POSTHOG_API_KEY = "${currentTeam?.api_token}"
+        let POSTHOG_HOST = "${window.location.origin}"
```

Comment:
I wonder if we've already got a function somewhere that does this 🤔 

Maybe in the login redirect logic 🤔 

(at the moment at least)

 - EU Cloud - API and UI host are the same
 - US Cloud - API is app.posthog.com UI is us.posthog.com
 
But,

if they set us.posthog.com then we correct it

https://github.com/PostHog/posthog-js/blob/b79315b7a4fa0caded7026bda2fec01defb0ba73/src/posthog-core.ts#L1742

(should probably not tell people to use incorrect value though)

Reply:
Not yet and I wondered exactly that.
https://posthog.slack.com/archives/C0460J93NBU/p1706544626918109

Since `window.location.origin` is already being used everywhere, I will merge this PR as it is and fix this in another PR, this would need to be done for every onboarding page.

is that ok?